concepts/05 Widgets/Drawer/Getting Started with Navigation Drawer/00 Create the Drawer.md

```diff
@@ -173,6 +173,30 @@ In addition, you can specify the [minSize](/api-reference/10%20UI%20Widgets/dxDr
     }
     export default App;
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+    <div>
```

Comment:
it's not neccessary

Reply:
There should be 1 root element in a template: https://github.com/vuejs/vue/issues/7088 - Allow more than 1 root element for Template

So, we create it in our demos:
https://js.devexpress.com/Demos/WidgetsGallery/Demo/Drawer/LeftOrRightPosition/Vue/Light/
<template>
  <div>
    <DxToolbar :items="toolbarContent"/>

https://js.devexpress.com/Demos/WidgetsGallery/Demo/Accordion/Overview/Vue/Light/
<template>
  <div id="accordion">
    <DxAccordion

And, it's very easy to copy-paste our code snippet to another project and add a new markup without wrapping all into a '1 root element': explicit <div> will help avoid some cases of this stupid mistake.

Reply:
Yes, exactly.

Reply:
https://github.com/DevExpress/devextreme-documentation/pull/447/commits/6ca95dbc559ad2a14c7775de3e636da197ebbbc1